src/dotnet-publish-iis/project.json

```diff
@@ -14,12 +14,21 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
 
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
```

Comment:
I don't believe there is anything special about tools here. Until recently portable apps needed to use netstandard1.5 but with the latest cli tools netcoreapp1.0 should be the tfm.